ovh/provider.go

```diff
@@ -73,6 +74,8 @@ func Provider() *schema.Provider {
 			"ovh_cloud_network_private":                                   resourceCloudNetworkPrivate(),
 			"ovh_cloud_network_private_subnet":                            resourceCloudNetworkPrivateSubnet(),
 			"ovh_cloud_user":                                              resourceCloudUser(),
+			"ovh_cloud_kubernetes_cluster":                                resourceCloudKubernetesCluster(),
+			"ovh_cloud_kubernetes_node_pool":                              resourceCloudKubernetesNodePool(),
```

Comment:
Please keep it sorted
```suggestion
			"ovh_cloud_kubernetes_cluster":                                resourceCloudKubernetesCluster(),
			"ovh_cloud_kubernetes_node_pool":                              resourceCloudKubernetesNodePool(),
			"ovh_cloud_network_private":                                   resourceCloudNetworkPrivate(),
			"ovh_cloud_network_private_subnet":                            resourceCloudNetworkPrivateSubnet(),
			"ovh_cloud_user":                                              resourceCloudUser(),
```